tests/inputFields.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+  //1. Select the PET TYPES menu item in the navigation bar
+  await page.getByText('Pet Types').click()
+  //2. Add assertion of the "Pet Types" text displayed above the table with the list of pet types
+  expect(await page.locator('h2').getByText('Pet Types').isVisible()).toBeTruthy()
+})
+
+test('Update pet type', async ({page}) => {
+  test.slow()
```

Comment:
I would assume that using this method you was trying to eliminate slowness of the test application on startup. This is because Petlinic is hosted on the cloud and if nobody use it, it goes to sleep and takes about 30-40 seconds to spin up backend API on first request.
But, this setting does not work and test failing at the next line, because you have configured actionTimeout of 10 seconds, which is a "bottle neck" in this situation.
`test.slow()` increases test timeout in 3 times, but **not** the action timeout. So as a result this command does not serve the purpose.
I see that your configuration file already have 90 seconds for the test timeout. This should be plenty even for the "cold start" of the application. So you can safely delete all `test.slow()` for the code and delete actionTimeout from configuration